src/views/v2/tickets/item.php

```diff
@@ -66,18 +66,21 @@
 	<?php endif; ?>
 >
 
-	<?php $this->template( 'v2/modal/item/remove', $context ); ?>
+	// @todo Convert this into an action.
+	<?php //$this->template( 'v2/modal/item/remove', $context ); ?>
 
 	<?php $this->template( 'v2/tickets/item/content', $context ); ?>
 
 	<?php $this->template( 'v2/tickets/item/quantity', $context ); ?>
 
 	<?php $this->template( 'v2/tickets/item/quantity-mini', $context ); ?>
 
-	<?php $this->template( 'v2/modal/item/total', $context ); ?>
+	// @todo Convert this into an action.
+	<?php //$this->template( 'v2/modal/item/total', $context ); ?>
 
 	<?php $this->template( 'v2/tickets/item/opt-out', $context ); ?>
 
-	<?php $this->template( 'v2/modal/item/opt-out', $context ); ?>
+	// @todo Convert this into an action.
+	<?php //$this->template( 'v2/modal/item/opt-out', $context ); ?>
```

Comment:
* :warning: This comment is 67% valid code; is this commented out code?
* :no_entry_sign: No space found before comment text; expected "// $this->template( 'v2/modal/item/opt-out', $context );" but found "//$this->template( 'v2/modal/item/opt-out', $context );"

<sub>hash: 560748faabf5cd587a2226058ee9c088</sub>